mobile/src/screens/main/home.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import { View } from '#libs/components/components';
+
+const Home: React.FC = () => {
+  return <View />;
```

Comment:
I can suggest returning null instead of an empty <View>.This approach ensures that the component is ignored in the component tree, meaning an unnecessary View won't be created. However, I understand that this file might change later, so I don't think you need to make any changes now. Just a tip for the future)

Reply:
please fix this comment)